test/acceptance/typography.gspec

```diff
@@ -47,9 +42,6 @@
 			css font-family matches "(?i)['|\"]apple sd gothic neo['|\"],\\s*['|\"]*dotum['|\"]*,\\s*sans-serif"
 		lang-japanese:
 			css font-family matches "(?i)['|\"]hiragino kaku gothic pro['|\"],\\s*['|\"]*meiyro['|\"]*,\\s*sans-serif"
-		&all:
```

Comment:
This was a pretty expensive set of tests (time-wise), so I pulled it into "paragraph".